owapi/util.py

```diff
@@ -167,3 +165,9 @@ def sanitize_string(string):
     removed_nonalphanumeric = re.sub(r'\W', '', space_converted)
     underscore_normalized = re.sub(r'_{2,}', '_', removed_nonalphanumeric)
     return underscore_normalized.replace("soldier_76", "soldier76")  # backwards compatability
+
+def correct_plural_name(name: str, value):
```

Comment:
These look like magic numbers (5, 7).

Is this correct for every language? English, Korean, German, etc?

Reply:
Korean test:
https://playoverwatch.com/ko-kr/career/pc/Kermit-22729